docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,119 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Proof-of-Stake mining?**
+
+Besides Proof-of-Work (PoW) mining Decred has Proof-of-Stake (PoS) mining. In PoS mining the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS pool and are randomly called upon to vote on the previous block. Every block 5 tickets are chosen from the pool and if at least 3 of those validate the previously mined block, the newly mined block is approved, added to the chain and both PoW miners and PoS miners are paid. If the block is not validated by the PoS miners the PoW miners do not get paid, but the PoS miners do. This is to incentivize PoW miners to mine according to the wishes of the PoS miners.
```

Comment:
"and are randomly called upon"

This sounds like the ticket will be called upon eventually, while that's not the case.  There is a slim chance that the lottery won't choose the ticket at all, in which case it becomes expired and must be revoked.

I know you cover this later in detail but this intro needs to be a bit more accurate.